core/src/epicli/data/common/ansible/playbooks/roles/node_exporter/tasks/install-node-exporter-as-daemonset.yml

```
@@ -9,6 +9,8 @@
     helm_chart_values_bool: "{{ specification.helm_chart_values | default(false) | ternary(true, false) }}"
 
   delegate_to: localhost
+  environment:
+   KUBECONFIG: "{{ vault_location }}/../kubeconfig"
```

```suggestion
   KUBECONFIG: "{{ kubeconfig.local }}"
```
How about putting it here: https://github.com/rpudlowski93/epiphany/blob/feature/1673/core/src/epicli/data/common/ansible/playbooks/node_exporter.yml ? :thinking:

---

Please notice using kubeconfig.local and inventory_dir will work in this case. :+1: